apps/frontend/src/pages/onboarding/onboarding.tsx

```diff
@@ -132,19 +138,25 @@ const Onboarding: React.FC = () => {
 								);
 							})}
 							<div className={styles["button-container"]}>
-								{!isLastQuestion && (
+								{!isLastQuestion && !isFirstQuestion && (
```

Comment:
```suggestion
								{!isFirstQuestion && (
```
We should display Back button for the last question